app/assets/javascripts/controllers/openshift_deployment/openshift_deployment_controller.js

```diff
@@ -0,0 +1,10 @@
+miqHttpInject(angular.module('openshiftDeployment', ['ui.bootstrap', 'patternfly', 'patternfly.charts', 'miq.card']))
```

Comment:
A lot of stuff are copied from the dashboard, ill take stuff down once ill figure out what i need.
no mockups yet, just trying to build a skeleton.